components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -146,6 +137,20 @@ - (void)presentationTransitionWillBegin {
   // presentation and dismissal of the dialog.
   // https://material.io/guidelines/motion/choreography.html#choreography-creation
 
+  // Ensure corner radius matches between the tracking view and the view being presented
+  if (self.dialogCornerRadius < 0) {
+    // If dialogCornerRadius is not set, use the presented view's cornerRadius for the shadow layer.
+    _trackingView.layer.cornerRadius = self.presentedViewController.view.layer.cornerRadius;
```

Comment:
This changes the behavior to be: if `dialogCornerRadius` is not set, then force cornerRadius to be the view's layerCornerRadius. If this behavior is okay, I am fine with the change.

Reply:
Cool. It sounds good if that's what UIKit suggests.